server/routes/alert.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import alertCtrl from '../controllers/alert';
+
+const router = express.Router(); // eslint-disable-line new-cap
+
+router.route('/')
+  .post(alertCtrl.sendPage);
```

Comment:
I was thinking about this a little bit and it seems like we could create an admin user for the queue. Seems a little janky, but for now I think it's fine. What do you think?